esmvalcore/cmor/_fixes/native6/era5.py

```diff
@@ -334,6 +334,26 @@ def fix_metadata(self, cubes):
         return cubes
 
 
+class Uas(Fix):
+    """Fixes for uas."""
+
+    def fix_metadata(self, cubes):
+        """Fix metadata."""
+        for cube in cubes:
+            fix_hourly_time_coordinate(cube)
+        return cubes
```

Comment:
Shifting the hourly time points back by half an hour is done for variables that are accumulations because for ERA5 data those represent the past hour and in CMOR the convention is to put the time point at the center of the interval instead of at the end. However, for instantaneous variables like the 10 metre U/V wind component this does not apply. The time point represents the moment at which the variable had this value, so this fix should not be used. You can read more about it here: https://confluence.ecmwf.int/display/CKB/ERA5%3A+data+documentation

Reply:
cheers @bouweandela - I didn't know this! :beer:

Reply:
@bouweandela Oh, good to know, thanks! A question though, why do we do a 30 mins down for `tasmax`, `tasmin`? By the looks of it "reanalysis: the minimum or maximum values are in the hour (the processing period) ending at the validity date/time", so as far as I understand, the validity is at the end of timestep and it's similar what you are saying for CMOR.

Reply:
`tasmin` and `tasmax` are always related to some interval in time. Where the time point is located does not really matter, what matters is the bounds. The developers of ERA5 chose to put the time point at the end of the interval, while for CMIP data it is customary to put the time point in the middle of the interval. For easier comparison of ERA5 with CMIP data, we shift the time point to the middle of the interval when that is possible without sacrificing the validity of the data.

Reply:
Thanks! Removed the fix for `Uas` and `Vas` :smile: Sorry, I caused that issue!